js/ui/widget/ui.errors.js

```diff
@@ -203,6 +203,31 @@ module.exports = errorUtils(errors.ERROR_MESSAGES, {
     */
     E1049: `Column '{0}': filtering is allowed but the 'dataField' or 'name' option is not specified`,
 
+    /**
+     * @name ErrorsUIWidgets.E1050
+     */
+    E1050: "The Quill script is referenced after the DevExtreme scripts or not referenced at all",
+
+    /**
+     * @name ErrorsUIWidgets.E1051
+     */
+    E1051: "The Quill-Delta-to-Html script is referenced after the DevExtreme scripts or not referenced at all",
```

Comment:
Would it be better to use a single parameterized error for these cases?
```The required {0} script is referenced after the DevExtreme scripts or not referenced at all?```

/cc @RomanTsukanov  @TikhonovaTatyana

Reply:
You're right. We can modify the already-existing E1041 error, which has JSZip hardcoded in it.